### ISSUE TITLE
This plugin generated a warning

### ISSUE THREAD

**Issue by nebosphere19** [merdok/homebridge-webos-tv#351]
This plugin generated a warning from the characteristic 'Volume': characteristic was supplied illegal value: number -1 exceeded minimum of 0.

This began after upgrading homebridge to the latest version: 1.3.1

Node.js Version | v14.15.4
-- | --
NPM Version | v7.5.4

Config:

```
{
    "devices": [
        {
            "name": "Playroom TV",
            "ip": "10.0.0.74",
            "mac": "EC:F4:51:4C:D4:76",
            "prefsDir": "~/.webosTv",
            "pollingInterval": 1,
            "legacyTvService": false,
            "showInputButtons": false,
            "volumeControl": "false",
            "volumeLimit": 100,
            "channelControl": false,
            "mediaControl": false,
            "infoButtonAction": "INFO"
        }
    ],
    "platform": "webostv"
}
```

**Comment by merdok** [comment]
You would need to start homebridge in debug mode and ideally enable deepDebugLog, then try to replicate the issue. After that please post the log snippet here so i can see what is happening before and after that warning appears.

**Comment by nebosphere19** [comment]
Here's what I could get. Let me know if you need more!


`[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Connected to TV
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Requesting TV information
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] system service - getSystemInfo requested. 
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] com.webos.service.update service - getCurrentSWInformation requested. 
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] api service - getServiceList requested. 
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] com.webos.applicationManager service - listLaunchPoints requested. 
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] tv service - getExternalInputList requested. 
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] com.webos.applicationManager service - listApps requested. 
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Retrieved launch points (inputs, apps)
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Retrieved all apps
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Retrieved SW info
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Retrieved service list
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Retrieved system info
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Retrieved external input list
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Got TV information
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] TV webOS version: 3.5
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Subscribing to TV services
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Subscribing to getPowerState from power service.
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Subscribing to getStatus from audio service.
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Subscribing to getForegroundAppInfo from com.webos.applicationManager service.
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Subscribing to getCurrentChannel from tv service.
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Subscribing to getSoundOutput from audio service.
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Subscribing to listLaunchPoints from com.webos.applicationManager service.
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] TV power status changed, status:  state: Active,
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] TV turned on!
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] App launched, current appId: com.webos.app.hdmi3
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Got new launch points list from tv!
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Audio status changed. Current volume: -1, Muted: No
[2/24/2021, 7:06:56 PM] [homebridge-webos-tv] This plugin generated a warning from the characteristic 'Volume': characteristic was supplied illegal value: number -1 exceeded minimum of 0. See https://git.io/JtMGR for more info.
[2/24/2021, 7:06:56 PM] [homebridge-webos-tv] Error: 
    at Volume.Characteristic.characteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2164:105)
    at Volume.Characteristic.validateUserInput (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2114:14)
    at Volume.Characteristic.updateValue (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1365:20)
    at webosTvDevice.updateTvAudioStatus (/usr/local/lib/node_modules/homebridge-webos-tv/index.js:1354:97)
    at LgTvController.<anonymous> (/usr/local/lib/node_modules/homebridge-webos-tv/index.js:182:12)
    at LgTvController.emit (events.js:315:20)
    at /usr/local/lib/node_modules/homebridge-webos-tv/lib/LgTvController.js:375:12
    at Object.f0014f6f0008 (/usr/local/lib/node_modules/homebridge-webos-tv/lib/LgTvController.js:630:15)
    at WebSocketConnection.<anonymous> (/usr/local/lib/node_modules/homebridge-webos-tv/node_modules/lgtv2/index.js:149:48)
    at WebSocketConnection.emit (events.js:315:20)
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Sound output changed. Current sound output: external_speaker
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Failed to subscribe to the channel service. It seems Live TV is not running, will retry after switching to Live TV.
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Subscribed to TV services
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Connecting to remote input socket
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Connected to remote pointer input socket
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] Setup finished
[2/24/2021, 7:06:56 PM] [webostv] [Playroom TV] TV setup finished, ready to control tv
[2/24/2021, 7:06:59 PM] [homebridge-webos-tv] This plugin generated a warning from the characteristic 'Volume': characteristic was supplied illegal value: number -1 exceeded minimum of 0. See https://git.io/JtMGR for more info.
[2/24/2021, 7:06:59 PM] [homebridge-webos-tv] Error: 
    at Volume.Characteristic.characteristicWarning (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2164:105)
    at Volume.Characteristic.validateUserInput (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2114:14)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1517:24
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/util/once.ts:9:18
    at webosTvDevice.getVolume (/usr/local/lib/node_modules/homebridge-webos-tv/index.js:939:5)
    at Volume.emit (events.js:315:20)
    at /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1493:14
    at new Promise (<anonymous>)
    at Volume.<anonymous> (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1491:12)
    at step (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:143:27)`

**Comment by merdok** [comment]
What volume level does it show on the tv?

**Comment by nebosphere19** [comment]
The tv is set to output audio to my sound bar via optical and the output
level is fixed. The volume on the tv itself doesn’t change.

**Comment by merdok** [comment]
Ok, so the tv is not in control of the volume anymore in that setup and this is probably the reason why the tv returns -1 as volume level. 

Still curious, what happens when you try to change the volume on your tv using the remote control in that case? Normally the volume changes and a zu overlay is shown on the tv, what happens in your case?

**Comment by nebosphere19** [comment]
The TV knows it’s connected to a sound bar and that the remote is controlling that so it displays a little plus and minus simple on the right side of the screen in the middle. It’s pretty small, no big overlay.

**Comment by merdok** [comment]
Ok, got it! Thanks for letting me know! Will fix the warning.

**Comment by nebosphere19** [comment]
Thanks!!

**Comment by merdok** [comment]
Should be fixed in version 2.1.0
